pkg/shp/cmd/build/run.go

```diff
@@ -19,8 +23,14 @@ import (
 type RunCommand struct {
 	cmd *cobra.Command // cobra command instance
 
+	ioStreams       *genericclioptions.IOStreams // io-streams instance
+	pw              *reactor.PodWatcher          // pod-watcher instance
+	logTail         *tail.Tail                   // follow container logs
+	tailLogsStarted map[string]bool              // controls tail instance per container
+
 	buildName    string                      // build name
 	buildRunSpec *buildv1alpha1.BuildRunSpec // stores command-line flags
+	Follow       bool                        // flag to tail pod logs
```

Comment:
nit: this probably should be a private attribute.

Reply:
fwiw this was not an oversight, but explicit choice based on what they do in `oc`

for example, https://github.com/openshift/oc/blob/master/pkg/cli/startbuild/startbuild.go#L94-L138
https://github.com/openshift/oc/blob/master/pkg/cli/cancelbuild/cancelbuild.go#L63-L77

and a bunch more...admittedly, I did not fully research how this pattern manifested

but we'll see how `shp` evolves, and sure, perhaps we break ranks here and stay consistent in the other direction